packages/functions/e2e/functions.e2e.js

```diff
@@ -51,6 +51,24 @@ describe('functions()', function () {
       const response = await functionRunner();
       response.data.should.equal(region);
     });
+
+    it('accepts passing in a custom url string as first arg to an app', async function () {
+      const customUrl = 'https://us-central1-react-native-firebase-testing.cloudfunctions.net';
+      const functionsForRegion = firebase.app().functions(customUrl);
```

Comment:
Nit but this isn't a function for region, so should name it different

Reply:
Updated 👍

Reply:
verified